docs/python/environments.md

```diff
@@ -23,14 +23,14 @@ This command automatically looks for and displays a list of available Python int
 The extension looks for interpreters in the following locations:
 
 - Standard paths such as `/usr/local/bin`, `/usr/sbin`, `/sbin`, `c:\\python27`, `c:\\python36`, etc.
-- Virtual environments located under the workspace (project) folder, unless the `python.terminal.activateEnvironments` setting is `false`, in which case virtual environments are ignored.
+- Virtual environments located under the workspace (project) folder.
 - Conda environments
 
 Selecting an interpreter from the list configures your User [Settings](/docs/getstarted/settings.md) accordingly. The Status Bar shows the current interpreter and when selected brings up a list of available interpreters. The Status Bar also reflects when no interpreter is selected.
 
 ![No interpreter selected](images/environments/no-interpreter-selected-statusbar.png)
 
-The currently-selected interpreter is applied when right-clicking a file and selecting **Run Python File in Terminal**. You can also use **Python: Create Terminal** to create a terminal with the current environment.
+The currently-selected interpreter is applied when right-clicking a file and selecting **Run Python File in Terminal**. You can also use **Python: Create Terminal** to create a terminal with the current environment activated. Note that setting `python.terminal.activateEnvironments` to `false` disallows activating virtual and conda environments with these commands.
```

Comment:
"disallows" is a little misleading as you can do it manually, we just don't do it on your behalf (basically the setting just means "run this one command first, then do what you would normally do", so setting it to false just skips running that initial command). I would say "skips" is probably a more accurate description.

Reply:
OK